jwst/cube_build/cube_build_step.py

```diff
@@ -38,7 +39,7 @@ class CubeBuildStep (Step):
 
     spec = """
          channel = option('1','2','3','4','all',default='all') # Channel
-         band = option('short','medium','long','short-medium','short-long','medium-short', \
+         band = option('short','medium','long','short-medium','short-long','medium-short',
```

Comment:
Testing on your branch now to look at the schema file.  This change is causing a parsing error - restoring the backslash at the end of the first line lets the step run.